go.mod

```diff
@@ -3,9 +3,13 @@ module github.com/gardener/gardener
 go 1.13
 
 require (
+	contrib.go.opencensus.io/exporter/ocagent v0.4.12 // indirect
```

Comment:
Can you revert the `go.mod` changes. Probably you are using golang@1.12? If yes, can you update to golang@1.13?

Reply:
thx, done